backend/accounts.go

```diff
@@ -25,11 +25,9 @@ import (
 
 	"github.com/BitBoxSwiss/bitbox-wallet-app/backend/accounts"
 	"github.com/BitBoxSwiss/bitbox-wallet-app/backend/accounts/types"
-	accountsTypes "github.com/BitBoxSwiss/bitbox-wallet-app/backend/accounts/types"
 	"github.com/BitBoxSwiss/bitbox-wallet-app/backend/bitsurance"
 	"github.com/BitBoxSwiss/bitbox-wallet-app/backend/coins/btc"
 	"github.com/BitBoxSwiss/bitbox-wallet-app/backend/coins/coin"
-	coinpkg "github.com/BitBoxSwiss/bitbox-wallet-app/backend/coins/coin"
```

Comment:
I think in case of dups, you should keep the named one and remove the unnamed one. The reason for naming these imports is to avoid clashes or generic names. `func foo(coin coin.Coin)` is no good :sweat_smile: 

Not sure what the idiomatic way is, as I assume renaming imports is not very idiomatic. I am afraid it might be one letter var name like `func foo(c coin.Coin)`, but I hate short names with a passion.

Reply:
> I think in case of dups, you should keep the named one and remove the unnamed one. The reason for naming these imports is to avoid clashes or generic names. `func foo(coin coin.Coin)` is no good 😅
> Not sure what the idiomatic way is, as I assume renaming imports is not very idiomatic.

Exactly, according to this: https://google.github.io/styleguide/go/decisions#import-renaming imports shouldn't be renamed unless there are good reasons

> The reason for naming these imports is to avoid clashes or generic names. `func foo(coin coin.Coin)` is no good 😅

I don't think this is solved by renaming the package import:
`func foo (coin coinpkg.Coin)` doesn't look much butter imho. In general having a type `Coin` in a package `coin` will inevitably leads us to situations like these.

Reply:
> `func foo (coin coinpkg.Coin)` doesn't look much better imho

I should have been more specific. It's not the optics, it's about the name shadowing. Inside the `foo` function you can't use the `coin` package anymore b/c it's shadowed by the `coin` argument. It's a bit silly that vars and packages live in the same namespace in Go, but since it is, we should not shadow.

Reply:
I'd personally argue that it is better to rename the argument rather than the package (smaller scope, and it's easier to find a more meaningful name for an arg rather than a pkg) but I will not fight over it :D I'll make the changes!

Reply:
It's not a fight :smile: I think your point makes sense and I'm okay with it too.